circle.yml

```diff
@@ -15,10 +15,11 @@ dependencies:
     - "~/.cabal"
 
   override:
+    - rm -rf /home/ubuntu/.go_workspace/src
```

Comment:
This seems unnecessary, CircleCI caches the `GOPATH` and this will negatively impact that